types/natural/index.d.ts

```diff
@@ -127,6 +127,7 @@ export function DamerauLevenshteinDistance(
 declare function DiceCoefficient(str1: string, str2: string): number;
 
 interface Stemmer {
+    removeStopWords(moreStopWords: string[]): void;
     stem(token: string): string;
```

Comment:
Prefer source order.
```suggestion
    stem(token: string): string;
    removeStopWords(moreStopWords: string[]): void;
```